catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -23,7 +23,7 @@
 
         <AD id="imageryProviders"
             name="Imagery Providers"
-            description='List of imagery providers to use. Valid types are: OSM (OpenStreetMap), AGM (ArcGisMap), BM (BingMap), WMS (WebMapService), WMT (WebMapTile), TMS (TileMapService), GE (GoogleEarth). Example: {"name": "Example OSM", "show": true, "type": "OSM", "url": "http://a.tile.openstreetmap.org", "fileExtension": "png", "order": 0, "alpha": 1, "proxyEnabled": false}.'
+            description='List of imagery providers to use. Valid types are: OSM (OpenStreetMap), AGM (ArcGisMap), BM (BingMap), WMS (WebMapService), WMTS (WebMapTileService), TMS (TileMapService), GE (GoogleEarth). Example: {"name": "Example OSM", "show": true, "type": "OSM", "url": "http://a.tile.openstreetmap.org", "fileExtension": "png", "order": 0, "alpha": 1, "proxyEnabled": false}.'
```

Comment:
We may want to leave this one; the imageryProviders are set in JSON in this field and i think for it to work correctly it needs to be WMT

Reply:
Ok. You're just saying this is preferable due to the fact that the end-user might get confused and write WMTS when the internal mapping config reads WMT, right?

Reply:
yes sir